recipe/meta.yaml

```diff
@@ -17,15 +17,16 @@ build:
 requirements:
   host:
     - pip
-    - python
+    - python >=3.5
     - setuptools
 
   run:
-    - python
+    - python >=3.5
```

Comment:
- python >=3.5
    - numpy >=1.9
    - scipy >=1.0
    - numba >=0.39
    - astropy >=3.1,<4
    - matplotlib >=2.0,!=3.0.1
    - plotly >=3.6,<4

Reply:
Can you try this combination?

Reply:
I have incorporated the changes